jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -519,3 +526,26 @@ def _live_update(self, event={}):
         for mark in self.marks.values():
             mark.update_xy(sp.spectral_axis.value, sp.flux.value)
             mark.visible = True
+
+    def translate_units(self, collapsed_spec):
+        sb_to_flux = {
+            u.MJy / u.sr: u.MJy / u.pix,
+            u.Jy / u.sr: u.Jy / u.pix,
+            u.erg / (u.cm**2*u.s*u.AA*u.sr): u.erg / (u.cm**2*u.s*u.AA),
+            u.erg / (u.s*u.cm**2*u.sr): u.erg / (u.s*u.cm**2),
+            u.W / (u.m**2*u.Hz*u.sr): u.W / (u.m**2*u.Hz)
```

Comment:
The dictionary will probably have to go when translating to any sb/flux. If there's eventually a drop down for the user to select their desired units, the path forward for this becomes clear.